src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -222,7 +223,8 @@
                            ssl-config)
                         :client-auth :none
                         :cipher-suites (or (:cipher-suites ssl-config) config/acceptable-ciphers)
-                        :protocols     (or (:protocols ssl-config) config/default-protocols)}))
+                        :protocols     (or (:protocols ssl-config) config/default-protocols)
+                        :cache-session-id (get ssl-config :cache-session-id true)}))
```

Comment:
Does anyone have any thoughts on what to do about this? Currently a user can configure `:cache-session-id` when adding a proxy route. This option didn't exist before for the proxy route configuration, but we also don't have a way to test it (from what I can tell, there's no way to access the SSL Context Factory for the proxy servlet from the test code, and a more substantial test has proved to be somewhat of a timesink). I'm wondering if we should just hardcode it to true here and use the default behavior?